runtimes/pythonrt/pythonrt_test.go

```diff
@@ -16,13 +16,14 @@ import (
 	"testing"
 	"time"
 
-	"go.autokitteh.dev/autokitteh/internal/kittehs"
-	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
-	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 	"go.uber.org/zap"
 	"golang.org/x/net/http2"
 	"golang.org/x/net/http2/h2c"
 
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+
```

Comment:
Where are all this formatting changes come from?

Reply:
AFAIK the Go formatter can handle correctly both 2-block and 3-block import sections, it just doesn't know that *our* default is 3 blocks. All the rest below is automatic for gofumpt.